elements/storytelling/src/components/editor.js

```diff
@@ -1,29 +1,22 @@
 import { LitElement, html } from "lit";
-import initEditor from "../helpers/editor";
+import "../../../jsonform/src/main.js";
```

Comment:
We shouldn't import eox-jsonform as this would increase bundle size. We should instead inform the developer that if the editor is used, also eox-jsonform needs to be installed.
See https://github.com/EOX-A/EOxElements/blob/main/elements/layercontrol/src/components/layer-config.js#L113-L115 for an example how this is handled in eox-layercontrol.

Reply:
Ahh, I didn't know we were informing the developer about the import of other `eox-elements`. Thanks for informing me, will look into this in future as well.

Reply:
Resolved ✅